python_examples/covidsim_latent_period_distribution.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
As useful as this notebook was for getting to understand the algorithm, there's no need to add it to the repository, especially as it is outdated.